etc/picongpu/hemera-hzdr/gpu.tpl

```diff
@@ -63,7 +63,7 @@
 
 # number of cores to block per GPU - we got 6 cpus per gpu
```

Comment:
Probably makes sense to update the comment as well. However, I'm not sure to what, as I don't understand why 3 is the correct value (but it really is).

Reply:
change it to: `number of non hyper thread cores to block per GPU - we got 3 cores per gpu`

ATTENTION: before it was `6 cpus` but it must be `cores`

Reply:
That is exactly what I do not understand: each node has 2x 12-core CPUs, so there are 6 physical ("non hyperthreaded") cores per GPU on the machine. Ofc now it is more important to just fix and not make it so I understand the fix.